### PR TITLE
Template css and js

### PR DESCRIPTION
**Pull request by ghost**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Task ID: TI-79

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: public/css/flexslider.css [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+@media screen and (max-width: 860px) {
+  .flex-direction-nav .flex-prev {
+    opacity: 1;
+    left: 10px;
+  }
+  .flex-direction-nav .flex-next {
+    opacity: 1;
+    right: 10px;
+  }
+}
```
Comment (at-nale) [inline public/css/flexslider.css]: thiếu endline

File: public/css/owl.theme.default.min.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    -ms-transition:opacity 200ms ease;
+    -o-transition:opacity 200ms ease;
+    transition:opacity 200ms ease;
+    -webkit-border-radius:30px;
+    -moz-border-radius:30px;
+    border-radius:30px
+}
+.owl-theme .owl-dots .owl-dot.active span, .owl-theme .owl-dots .owl-dot:hover span {
+    background:#869791
+}*/
```
Comment (at-nale) [inline public/css/owl.theme.default.min.css]: thiếu endline

File: public/js/google_map.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                position: latlng,
+                map: map,
+                icon: 'images/loc.png'
+            });
+
+        });
+    }
+    
+}
+google.maps.event.addDomListener(window, 'load', init);
```
Comment (at-nale) [inline public/js/google_map.js]: thiếu endline

File: public/js/jquery.countTo.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+      if (init) {
+        if (data) data.stop();
+        $this.data('countTo', data = new CountTo(this, options));
+      }
+
+      data[method].call(data);
+    });
+  };
+}));
```
Comment (at-nale) [inline public/js/jquery.countTo.js]: cũng như trên

File: public/js/jquery.easing.1.3.js [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *  COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ *  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE
+ *  GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED 
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ *  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED 
+ * OF THE POSSIBILITY OF SUCH DAMAGE. 
+ *
+ */
```
Comment (at-nale) [inline public/js/jquery.easing.1.3.js]: như trên

File: public/js/magnific-popup-options.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+	
+
+
+	// Call the functions 
+	magnifPopup();
+	linkMagnifPopup();
+	magnifVideo();
+
+});
```
Comment (at-nale) [inline public/js/magnific-popup-options.js]: như trên

File: public/js/main.js [hunk 7]
```diff
@@ -2,3 +2,314 @@ $('.logout').click(function(event) {
...
+	var datePicker = function() {
+		// jQuery('#time').timepicker();
+		jQuery('.date').datepicker({
+		  'format': 'm/d/yyyy',
+		  'autoclose': true
+		});
+	};
+
+
+
```
Comment (at-nale) [inline public/js/main.js]: xóa  khoảng trống

File: public/js/main.js [hunk 8]
```diff
@@ -2,3 +2,314 @@ $('.logout').click(function(event) {
...
+		sliderMain();
+		dropdown();
+		goToTop();
+		loaderPage();
+		owlCrouselFeatureSlide();
+		parallax();
+		datePicker();
+	});
+
+
```
Comment (at-nale) [inline public/js/main.js]: xóa khoảng trống

**Review by mvn-nale-dn — COMMENTED** [review]
LGTM

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**